Update 04-expertise.md

ErinBecker opened the pull request:
This PR makes several changes to episode 04-expertise.md, including:

1) Updating objectives, key points and questions to stick to a small number of each and fit the new episode contents.

2) Changing the episode title to focus on expertise (there is no discussion of memory in this episode anymore).

3) Adding explicit examples of how the limitations of expertise affect teaching in our workshop setting. 

4) Move content about dismissive language from motivation section because it ties in well with discussion of expert blind spot. 

5) Add exercises for understanding limitations of expertise and developing strategies to overcome them while teaching. 

6) Reiterate importance of formative assessment and explicitly say how it can help overcome limitations of expertise.

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -1,103 +1,120 @@
 from earlier stages of learning, how being an expert can make it 
 more difficult to teach novices, and some tools to help instructors 
-identify their own expert blind spots.  
+identify overcome these difficulties.  
```
+identify and overcome?

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -1,103 +1,120 @@
-An [earlier topic]({{ page.root }}/01-practice.md) described a key 
+An [earlier topic]({{ page.root }}/02-practice.md) described a key 
 difference between novices and competent practitioners. Competent practitioners 
 have mental models that work well enough for most situations. 
```
Add a note about novices: "Novices lack a mental model, and competent practitioners have..."
OR
Remove the "Competent practitioners have..." sentence.  

This sets up the "how are experts different from both" question more clearly.

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -1,103 +1,120 @@
-> ## Other Consequences of Expertise
+## Limitations of Expertise  
+
+Because your learners' mental models will be less densly connected than your own, 
```
Not all over our instructors are experts -- maybe add a "likely" or "probably"?  That waters down the language but...

Also, densly -> densely

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -1,103 +1,120 @@
-In fact, experts are frequently so familiar with their subject
+Experts are frequently so familiar with their subject
 that they can no longer imagine what it's like to *not* see the world that way. 
 This is called *expert blind spot* and can lead to what's known as the *expertise-reversal effect*.
```
Based on this text, it's not clear to me what the expertise reversal effect is.

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -111,15 +128,54 @@ reason why we welcome instructors who still identify as "novices"!  Someone
+> This discussion should take about 5 minutes.  
+>
+>> ## Solution  
+>> Courtney Seiter lists [10 words and phrases][motivation-words] that can change a conversation: if, could, yes, together, thank you,
```
italicize the words for emphasis?

ChristinaLK commented on _episodes/04-expertise.md:
```diff
@@ -150,43 +206,25 @@ For these reasons,
 > Many of the foundational concepts of computer science,
 > such as computability,
-> inhabit the lower-right corner of the grid described above.
+> are diffult to learn and not immediately useful.
```
diffult -> difficult

ErinBecker commented:
Thanks for the thorough review @ChristinaLK! I think I've addressed all your comments, including the addition of how we frame asking for questions. Great addition.

ErinBecker commented:
Merged at approval from @ChristinaLK
